packages/app/src/cli/models/app/app.ts

```diff
@@ -299,6 +304,21 @@ export class App implements AppInterface {
     if (isLegacyAppSchema(configuration)) return configuration
     return configuration as CurrentAppConfiguration & SpecsAppConfiguration
   }
+
+  private filterDeclarativeWebhooksConfig() {
+    const webhooksConfigIndex = this.realExtensions.findIndex((ext) => ext.handle === 'webhooks')
```

Comment:
just to note for myself, we'll probably have to add new filtering here for singular webhook modules 📓